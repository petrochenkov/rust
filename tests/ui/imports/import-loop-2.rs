mod a {
    pub use crate::b::x; //~ ERROR unresolved import `crate::b::x`
    //~^ ERROR cannot determine resolution for the import
}

mod b {
    pub use crate::a::x; //~ ERROR cannot determine resolution for the import

    fn main() { let y = x; }
}

fn main() {}
