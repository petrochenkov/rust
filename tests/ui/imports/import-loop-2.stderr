error: cannot determine resolution for the import
  --> $DIR/import-loop-2.rs:2:13
   |
LL |     pub use crate::b::x;
   |             ^^^^^^^^^^^

error: cannot determine resolution for the import
  --> $DIR/import-loop-2.rs:7:13
   |
LL |     pub use crate::a::x;
   |             ^^^^^^^^^^^

error[E0432]: unresolved import `crate::b::x`
  --> $DIR/import-loop-2.rs:2:13
   |
LL |     pub use crate::b::x;
   |             ^^^^^^^^^^^ no `x` in `b`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0432`.
